tests/tests_api.c

```diff
@@ -2733,9 +2737,6 @@ static void tests_echo_tfa(void)
     api_format_send_cmd(cmd_str(CMD_seed), "{\"source\":\"create\", \"key\":\"password\"}",
                         KEY_STANDARD);
     ASSERT_REPORT_HAS(flag_msg(DBB_ERR_IO_LOCKED));
-
-    api_format_send_cmd(cmd_str(CMD_backup), attr_str(ATTR_list), KEY_STANDARD);
-    ASSERT_REPORT_HAS(flag_msg(DBB_ERR_IO_LOCKED));
```

Comment:
better to leave this in and instead test for successful completion. 

also would be good to add a test that a backup can be verified successfully.

Reply:
done.

the check backup is covered by the hidden wallet tests, which is roughly the same. I tried to add it here too but it got time consuming and complicated quickly, so imho not worth adding a redundant test.